src/Compilers/CSharp/Portable/Parser/Lexer_StringLiteral.cs

```diff
@@ -626,19 +594,10 @@ private void ScanInterpolatedStringLiteralHoleBalancedText(char endingChar, bool
                             switch (_lexer.TextWindow.PeekChar(1))
                             {
                                 case '/':
-                                    _lexer.TextWindow.AdvanceChar(); // skip /
-                                    _lexer.TextWindow.AdvanceChar(); // skip /
-
-                                    // read up to the end of the line.
-                                    while (!IsAtEnd(allowNewline: false))
-                                    {
-                                        _lexer.TextWindow.AdvanceChar(); // skip // comment character
-                                    }
-
+                                    _lexer.ScanToEndOfLine();
                                     continue;
                                 case '*':
-                                    // check for and scan /* comment */
-                                    ScanInterpolatedStringLiteralNestedComment();
+                                    _lexer.ScanMultiLineComment(out _);
```

Comment:
can (and should) just reuse the actual helpers we have here.

Reply:
note: we have a *lot* of tests for this area (several of my previous PRs beefed this up a ton).